src/Makefile

```diff
@@ -401,8 +401,9 @@ check: test
 lcov:
 	$(MAKE) gcov
 	@(set -e; cd ..; ./runtest --clients 1)
-	@geninfo -o redis.info .
-	@genhtml --legend -o lcov-html redis.info
+	@geninfo -o KeyDB.info --no-external .
+	@genhtml --legend -o lcov-html KeyDB.info
```

Comment:
You'll have to update redis.info in more places than just this... e.g. the clean target.